rotkehlchen/tests/api/test_liquity.py

```diff
@@ -270,3 +270,21 @@ def test_staking_stats(rotkehlchen_api_server, ethereum_accounts):
     assert address_0_data['stability_pool_gains'][0]['amount'] == '1.7820064710306824'
     assert address_1_data['stability_pool_gains'][0]['amount'] == '12.294706987216218'
     assert address_1_data['stability_pool_gains'][1]['amount'] == '4240.34942308358'
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xbB8311c7bAD518f0D8f907Cad26c5CcC85a06dC4']])
+@pytest.mark.parametrize('ethereum_modules', [['liquity']])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+def test_proxy_info_is_shown(rotkehlchen_api_server, ethereum_accounts):
+    """Check that `proxy_to` is not None for proxy addresses."""
+    user_address = ethereum_accounts[0]
+    response = requests.get(api_url_for(
+        rotkehlchen_api_server,
+        'liquitystabilitypoolresource',
+    ), json={'async_query': False})
+
+    result = assert_proper_response_with_result(response)
+    assert not result[user_address]['proxy_to']  # since this address is the proxy owner
```

Comment:
Can you check instead

```suggestion
    assert result[user_address]['proxy_to'] is None # since this address is the proxy owner
```